src/StepAggregator.php

```diff
@@ -192,4 +159,35 @@ public function getName()
     {
         return $this->name;
     }
+
+    /**
+     * Builds the pipeline
+     *
+     * @return callable
+     */
+    private function buildPipeline()
```

Comment:
Should we create a `PipelineBuilder` which is responsible for creating a pipeline?